experiments/mp-game-6/game.core.js

```diff
@@ -227,7 +229,16 @@ game_core.prototype.makeTrialList = function (connection, callback) {
         "CONJUNCTION_DISJUNCTION",
         "DISJUNCTION_CONJUNCTION"
     ];
-
+    while(ruleTypes.length !== this.numRounds){
```

Comment:
Sort of discussed in our meeting today, but I think a "ground-up" approach to creating the array would be better.

```javascript
theseRuleTypes = [];
for (var i = 0; i < this.numRounds; i++) {
    var randomSelection = ruleTypes[Math.floor(...)]; // etc
    theseRuleTypes.push(randomSelection);
}
```

the splicing/modifying the array as you loop makes me nervous.